sympy/physics/control/lti.py

```diff
@@ -1656,31 +1656,61 @@ def __neg__(self):
         return MIMOParallel(*arg_list)
 
 
-class Feedback(Basic):
-    """
-    A class for representing negative feedback interconnection between two
-    input/output systems. The first argument, ``num``, is called as the
-    primary plant or the numerator, and the second argument, ``den``, is
-    called as the feedback plant (which is often a feedback controller) or
-    the denominator. Both ``num`` and ``den`` can either be ``Series`` or
+class Feedback(SISOLinearTimeInvariant):
+    r"""
+    A class for representing closed-loop feedback interconnection between two
+    SISO input/output systems. The first argument, ``plant``, is the
+    primary plant of the closed-loop system or in simple words, the transfer
+    function representing the system/process to be controlled. Most of the time, a controller system is also
+    placed in series with the plant. This controller is known as the **plant controller** as it controls
+    the plant by varying the input that goes into plant. The second argument, ``controller``,
+    as the name suggests, controls the fed back signal to the ``plant``. It is the equivalent transfer
+    function of the feedback controling system which is generally a sensor system that constantly
+    takes in the output signal produced and feeds it back to the primary plant (or a plant-controller
+    if one is connected in series with the plant). Note that the feedback controller is not same as the
+    plant controller.
+
+    Generally, in control theory, we want the output signal to follow the input signal or in
+    other words, the error signal (difference between the output signal and input signal) to
+    be as small as possible. This is the key purpose of the **negative** feedback loop. The output
+    signal produced by the feedback controller is subtracted from the reference (input) signal to produce
+    the error signal ("difference" is taken and hence the name **Negative Feedback**). The error signal
+    is fed back to the plant/plant-controller through the feedback loop and that in turn produces the
+    optimal output. In some systems, we feed back the sum of output signals from the feedback
+    controller and reference (input) signal to the plant. This type of feedback loop is known as the
+    positive feedback loop ("sum" is taken and hence the name **Positive Feedback**).
+
+    Both ``plant`` and ``controller`` can either be ``Series`` or
     ``TransferFunction`` objects.
 
     Parameters
     ==========
 
-    num : Series, TransferFunction
-        The primary plant.
-    den : Series, TransferFunction
-        The feedback plant (often a feedback controller).
+    plant : Series, TransferFunction
+        The primary plant. Also commonly known as the open
```

Comment:
Primary is misplaced here. plant has no significance. Also open-loop gain is `plant*controller` shown with `L` in many textbooks. Hence this needs correction.